mobile/examples/object_detection/ios/README.md

```diff
@@ -0,0 +1,48 @@
+# ONNX Runtime Mobile object detection iOS sample application
+
+This is an example app using object detection which is able to continuously detect the objects in the frames seen by your iOS device's back camera and display the detected object bounding boxes, detected class and corresponding inference confidence on the screen.
+
+This example is loosely based on [Google Tensorflow lite - Object Detection Examples](https://github.com/tensorflow/examples/)
+
+### Model
+We use pre-trained MobileNet SSD V2 (float) model from TensorFlow in this sample app. 
+[//]: # (Add the TF model link here)
+
+## Requirements
+- Install Xcode 12.5 and above (preferably latest version)
+- A valid Apple Developer ID
+- A real iOS device with a camera (preferably iphone 12/iphone 12 pro)
+- Xcode command line tools `xcode-select --install`
+- Clone the `onnxruntime-inference-examples` source code repo
+
+## Build And Run
+
+0. [Optional] Prepare ORT format model from TF -> ONNX -> ORT
+    - Open [Converting SSD Mobilenet from Tensorflow to ONNX](https://github.com/onnx/tensorflow-onnx/blob/master/tutorials/ConvertingSSDMobilenetToONNX.ipynb). This notebook will demostrate you how to convert pre-trained ssd models from TF to onnx.
+    - Once you get the model in onnx format, e.g. `ssd_mobilenet_v2_float.onnx`, you can follow [Convert ONNX models to ORT format](https://www.onnxruntime.ai/docs/how-to/mobile/model-conversion.html#converting-onnx-models-to-ort-format) to get an ORT format model.
+
+    Note: This step is optional. A converted ort format model is provided [here](https://microsoft-my.sharepoint.com/personal/guorachel_microsoft_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fguorachel%5Fmicrosoft%5Fcom%2FDocuments%2FShared%2FPublic) along with an onnx model format file.
```

Comment:
Maybe try the direct link without personal information

Reply:
should we consider putting it in one of the Azure storage accounts? not sure if we already have a place for example models, but maybe it would be good to have one.

Reply:
Yes. It would be better to put it in a more official host link.